### PR TITLE
Add cleanup job for plugins back

### PR DESCRIPTION
**Pull request by tduong992**

### CODE REVIEW

**Comment by tduong992** [comment]
TODO: test coverage

File: src/PluginCleanupInterface.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * @file
+ * Contains \Drupal\ultimate_cron\PluginCleanupInterface.
+ */
+
+namespace Drupal\ultimate_cron;
+
+interface PluginCleanupInterface {
+
+  /**
+   * Inform Ultimate Cron about cron jobs.
```
Comment (Berdir) [inline src/PluginCleanupInterface.php]: Documentation has been copied from hook_cronapi(), this needs to explain what this is about. Something like "Cleans and purges data stored by this plugin. The class should also have a docblock, something like "Plugins can implement this to be invoked for regular cleanup tasks."

File: ultimate_cron.module [hunk 2]
```diff
@@ -191,24 +192,14 @@ function ultimate_cron_help($route_name, \Drupal\Core\Routing\RouteMatchInterfac
...
-            '@type' => $info['defaults']['static']['title singular proper'],
-            '@title' => $plugin->title,
-          )),
-          'callback' => 'ultimate_cron_plugin_cleanup',
-          'callback arguments' => array(
-            'type' => $plugin_type,
-            'name' => $name,
-          ),
-        );
+      if ($plugin->isValid() && ($plugin instanceof \Drupal\ultimate_cron\PluginCleanupInterface)) {
```
Comment (Berdir) [inline ultimate_cron.module]: should use the interface and only use the short name here.

**Comment by Berdir** [comment]
Thanks, merged manually and cleaned up a bit before.

**Pull request closed without merging**